src/utils.py

```diff
@@ -533,7 +534,7 @@ def print_total(elems, name):
     elif total == 1:
         report("1 " + name)
     else:
-        report("%d %ss" % (total, name))
+        report("%d %ss \n" % (total, name))
```

Comment:
Logentries code:
If you see what is being printed. It just represents the name as plural.